public/index.ts

```diff
@@ -1033,19 +1034,24 @@ function main() {
     myState.showBoundingBox = !myState.showBoundingBox;
     view3D.setShowBoundingBox(myState.volume, myState.showBoundingBox);
   });
+
+  // convert value to rgb array
+  function hexToRgb(hex, last: [number, number, number]): [number, number, number] {
```

Comment:
I know you didn't actually add this (just moved it), but I'm surprised we don't already have this function somewhere in this repo. Doesn't `Color` have some built in function fo this?

Reply:
Yes, this app layer code could do the conversion in other ways, such as farming it out to threejs. 

We are putting the responsibility for this conversion outside of the main volume viewer, because the View3d interface wants a [number, number, number] for rgb color settings.  And we are avoiding having any explicit threejs type dependencies in the public api for View3d.  So the apps that embed this viewer have to do whatever color format conversions are needed.